BrainPortal/lib/act_rec_log.rb

```diff
@@ -272,7 +272,7 @@ def addlog_context(context, message=nil, caller_level=0)
     class_name     = context.class.to_s
     class_name     = context.to_s if class_name == "Class"
     rev_info       = context.revision_info
-    pretty_info    = rev_info.svn_id_rev
+    pretty_info    = rev_info.short_commit
```

Comment:
All these changes end up with an empty string being used instead of the short_commit (or other attributes) because the revision_info object is initially just a place holder with no data in it until `self_update()` is invoked. An example of that occurs when uploading a file and checking the file's log, we get:

```
[2016-08-08 13:13:25 EDT] create() UserfilesController rev.  Uploaded by admin
```

(see the two spaces after "rev." where the short commit is missing)

The current code base does not behave like the original (SVN compatible) one, where inquiring about any of the attribute values (commit, author, date etc) always fetched them from a self-updated object.